apps/snitch_core/lib/core/data/schema/variant.ex

```diff
@@ -4,60 +4,58 @@ defmodule Snitch.Data.Schema.Variant do
   """
 
   use Snitch.Data.Schema
-
   import Ecto.Query
 
   alias Snitch.Data.Schema.StockItem
-  alias Snitch.Repo
   alias Money.Ecto.Composite.Type, as: MoneyType
+  alias Snitch.Repo
 
   @type t :: %__MODULE__{}
 
   schema "snitch_variants" do
-    field(:sku, :string, default: "")
+    field(:sku, :string)
     field(:weight, :decimal, default: Decimal.new(0))
-    field(:height, :decimal)
-    field(:width, :decimal)
-    field(:depth, :decimal)
-    field(:is_master, :boolean, default: false)
+    field(:height, :decimal, default: Decimal.new(0))
+    field(:width, :decimal, default: Decimal.new(0))
+    field(:depth, :decimal, default: Decimal.new(0))
+    field(:selling_price, MoneyType)
     field(:cost_price, MoneyType)
     field(:position, :integer)
     field(:track_inventory, :boolean, default: true)
-    field(:discontinue_on, :naive_datetime)
+    field(:discontinue_on, :utc_datetime)
 
     has_many(:stock_items, StockItem)
 
     timestamps()
   end
 
-  @permitted_fields ~w(sku weight height width depth is_master)a ++
-                      ~w(cost_price position track_inventory discontinue_on)a
+  @cast_fields ~w(sku weight height width depth selling_price)a ++
+                 ~w(cost_price position track_inventory discontinue_on)a
```

Comment:
Can this be written as
`@cast_fields ~w(sku weight height width depth is_master cost_price position track_inventory discontinue_on)a`